src/os/kernel/kernel.c

```diff
@@ -87,6 +87,12 @@ uint kernel_create_process(void (*pointer_to_task_function)(void), int necessity
 	new_node->status = READY;
 	new_node->next = NULL;
 
+	tack_words += new_node.stack_size - 17;
```

Comment:
task_words doesn't seem to be defined anywhere else. I assume it is supposed to be stack_words, additionally, it has never been defined yet.